lib/content-services/content-node-selector/content-node-selector.service.spec.ts

```diff
@@ -0,0 +1,106 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { async, TestBed } from '@angular/core/testing';
+import { QueryBody } from 'alfresco-js-api';
+import { SearchService } from '@alfresco/adf-core';
+import { ContentNodeSelectorService } from './content-node-selector.service';
+
+class SearchServiceMock {
+    public query: QueryBody;
+    search(query: QueryBody) {
+        this.query = query;
+    }
+}
+
+describe('ContentNodeSelectorService', () => {
+
+    let service: ContentNodeSelectorService,
+        search: SearchServiceMock;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                ContentNodeSelectorService,
+                { provide: SearchService, useClass: SearchServiceMock }
+            ]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
```

Comment:
can we move it in the then of compileComponents?

Reply:
No.